.github/workflows/main.yml

```diff
@@ -47,24 +60,86 @@ jobs:
         with:
           extra_nix_config: 'access-tokens = github.com=${{ github.token }}'
 
+      - name: Set version 🏷️
+        run: |
+          echo ${{ needs.prepare.outputs.version }} > ./VERSION
+          echo $(git rev-parse HEAD) >> ./VERSION
+
       - name: Build installer 🛠️
         run: |
-          nix build '.#nixosConfigurations.mysystem.config.system.build.installer'
+          nix build -vL '.#nixosConfigurations.mysystem.config.system.build.installer'
 
       - name: Upload installer 📤
         uses: actions/upload-artifact@v3
         with:
           name: installer
           path: result/tarball/nixos-wsl-installer.tar.gz
 
+  launcher:
+    name: Build Launcher 🛠️
+    needs:
+      - prepare
+      - build
+    runs-on: windows-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Download installer
+        uses: actions/download-artifact@v3
+        with:
+          name: installer
+
+      - name: Setup MSBuild
+        uses: microsoft/setup-msbuild@v1.1
+        with:
+          vs-version: '[17.2,'
```

Comment:
Is this a version regex? Looks half complete to me.

Reply:
It's a range from (including) 17.2 to infinity